api/src/main/java/org/openmrs/api/context/UserPassCredentials.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.api.context;
+
+/**
+ * User-pass credentials represent the usual OpenMRS authentication credentials made of a pair username + password.
+ */
+public class UserPassCredentials implements Credentials {
```

Comment:
Can you put a since annotation on this?

Reply:
Any reason why this was not named UsernamePasswordCredentials or NamePasswordCredentials?